extensions/amp-image-slider/0.1/amp-image-slider.js

```diff
@@ -295,8 +296,13 @@ export class AmpImageSlider extends AMP.BaseElement {
    * @private
    */
   checkARIA_() {
-    const leftAmpImage = dev().assertElement(this.leftAmpImage_);
-    const rightAmpImage = dev().assertElement(this.rightAmpImage_);
+    if (!this.containsAMPImages_) {
+      return;
+    }
+
+    // Only if there are AMP-IMG Elements in use should this pathway execute.
+    const leftAmpImage = dev().assertElement(this.leftImage_);
```

Comment:
A bit odd to see two different conventions for AMP capitalization (`leftAmpImage` here or `AmpElement` vs `containsAMPImages` above). `this.containsAmpImages_` (while admittedly uglier) might align better

Reply:
Good call, resolving.